extensions/positron-python/scripts/pip-compile-ipykernel.py

```diff
@@ -1,3 +1,10 @@
+# /// script
+# requires-python = "==3.9.*"
+# dependencies = [
+#     "pip",
+#     "pip-tools",
+# ]
+# ///
```

Comment:
A small update so I can use `uv run scripts/pip-compile-ipykernel.py` haha

Reply:
I like it!